packages/service-content/src/lib/resources/queries/get-newsletter.ts

```diff
@@ -0,0 +1,32 @@
+import { sql } from '@blms/database';
```

Comment:
Maybe we could group getNewsletterQuery and getNewslettersQuery (and check if there is an id provided in params) instead ? Don't know if that makes sense, but that would prevent having to change both of them each time

What do you think @trigger67 ?